calendar-bundle/src/Resources/contao/modules/ModuleEventlist.php

```diff
@@ -401,7 +401,7 @@ protected function compile()
 				if (null !== $figure)
 				{
 					// Rebuild with link to event if none is set
-					if (!$figure->getLinkHref())
+					if (!$figure->getLinkHref() && $event['href'] != Environment::get('request'))
```

Comment:
This is only executed for the list module, right?
The reader module has its own code for the figure without the auto-link already: https://github.com/contao/contao/blob/f0bb2fd04dffad786828dd55c8ca592de3579c9d/calendar-bundle/src/Resources/contao/modules/ModuleEventReader.php#L324-L336

So I think this change is not needed.

```suggestion
					if (!$figure->getLinkHref())
```